core/src/main/java/com/yahoo/imapnio/async/data/FetchResult.java

```diff
@@ -0,0 +1,33 @@
+package com.yahoo.imapnio.async.data;
+
+import java.util.List;
+
+import javax.annotation.Nonnull;
+
+import com.sun.mail.imap.protocol.IMAPResponse;
+
+/**
+ * This class provides the list of IMAP response from fetch command response.
+ */
+public class FetchResult {
+
+    /** The collection of IMAP responses. */
+    private final List<IMAPResponse> imapResponses;
+
+    /**
+     * Initializes a {@code FetchResult} object with IMAP responses collection.
```

Comment:
Describes what kind of response lines will convert to FetchResponse object, what will not.  Give an example of each case.